types/three/OTHER_FILES.txt

```diff
@@ -0,0 +1,303 @@
+examples/jsm/animation/AnimationClipCreator.d.ts
```

Comment:
What are these?  Why are there 300?

Reply:
The `three` repo has two sections of its source code - `src` & `examples`. We're currently not able to write enough tests for every file, so @Methuselah96 made the decision to move the examples into `OTHER_FILES` as we still need them bunlded.

Reply:
Yeah, they're not included in the main export of the package, but they can be imported via their path (e.g., `import { AnimationClipCreator } from 'three/examples/jsm/animation/AnimationClipCreator';`).